pkg/kube/client.go

```diff
@@ -127,7 +127,7 @@ func (c *Client) Create(resources ResourceList) (*Result, error) {
 }
 
 // Wait up to the given timeout for the specified resources to be ready
-func (c *Client) Wait(resources ResourceList, timeout time.Duration) error {
+func (c *Client) Wait(resources ResourceList, timeout time.Duration, waitForJobsEnabled bool) error {
```

Comment:
`Wait` is part of the public API for the Go package. The signature cannot be changed until Helm v4. Can you please create a new function with a new name (e.g., `WaitWithJobs`). The old function can call the new function.

This has to do with Helm providing a Semantically Versioned API. It applies everywhere `Wait` was changed.

Reply:
Thanks for your detailed instructions. Done